.github/workflows/playwright-test-reusable.yml

```
@@ -0,0 +1,83 @@
+name: Playwright Tests - Reusable
+
+on:
+  workflow_call:
+    inputs:
+      test-mode:
+        description: 'Test mode: local (pnpm start from local), docker-build, or docker-pull'
+        required: false
+        default: 'local'
+        type: string
+      shards:
+        description: 'Shards for parallel execution'
+        required: false
+        default: '[1]'
+        type: string
+      docker-image:
+        description: 'Docker image to use (for docker-pull mode)'
+        required: false
+        default: 'n8nio/n8n:nightly'
+        type: string
+
+    secrets:
+      CURRENTS_RECORD_KEY:
+        required: true
+
+env:
+  PLAYWRIGHT_BROWSERS_PATH: packages/testing/playwright/ms-playwright-cache
+  NODE_OPTIONS: --max-old-space-size=4096
+  TESTCONTAINERS_RYUK_DISABLED: true # Disable Ryuk to avoid issues with Docker since it needs privileged access, containers are cleaned on teardown anyway
```

nit: can we move the comment above to have a bit shorter line
```suggestion
  # Disable Ryuk to avoid issues with Docker since it needs privileged access, containers are cleaned on teardown anyway
  TESTCONTAINERS_RYUK_DISABLED: true
```